source/UberRides/Model/PaymentMethod.swift

```diff
@@ -54,4 +54,11 @@ struct PaymentMethods: Codable {
         case methodID           = "payment_method_id"
         case type               = "type"
     }
+
+    public required init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        paymentDescription = try container.decodeIfPresent(String.self, forKey: .paymentDescription) ?? ""
```

Comment:
Any reason not to leave this optional?

Reply:
I had this in here when tweaking around, but it was easier to leave it in because otherwise I would have to add default values for class parameters. 

Since that was a bit confusing for code readability, I decided to add an explicit initializer in each one of these models instead of leaving them inferred.

Reply:
Changed to an optional. I also audited other instances of optionals vs. default strings and switched them to optionals.